manifests/kustomize/base/installs/multi-user/persistence-agent/params.env

```diff
@@ -0,0 +1 @@
+MULTIUSER=true
```

Comment:
Do we have a global config for this? Feels quite redundant to repeat this for all backend pods, especially when the  file is already under a path with "multi-user".

Reply:
> Do we have a global config for this? Feels quite redundant to repeat this for all backend pods, especially when the file is already under a path with "multi-user".

We just copied the style that your team used so far in https://github.com/kubeflow/manifests/blob/746c523e9e994154df48d4441d01a30e4f9c8de4/apps/pipeline/upstream/base/installs/multi-user/api-service/params.env#L1 . We could consider adding it here https://github.com/kubeflow/pipelines/blob/master/manifests/kustomize/base/installs/multi-user/params.yaml if you want.

Reply:
@chensun, @juliusvonkohout Currently with my addition, `MULTIUSER` is used by two services - **app_server** and **persistent_agent.** We could try to export it in a common params.env file but I would bring up a question about other env variables which are even more frequently used like - `KUBEFLOW_USERID_HEADER`, `KUBEFLOW_USERID_PREFIX`, `NAMESPACE`. Should we also export them? 
But all this is out of the scope of this PR. I would open a dedicated issue about this and let this PR get merged as it is.

Reply:
Agree it's out of scope for this PR.